schema/DoctrineMigrations/Version20230817114147.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230817114147 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Persist PublicEntities and AdministratorRelPublicEntities in PublicEntities db table';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql(
+            'INSERT INTO PublicEntities
+                (iden, fqdn, platform, brand, client, name_en, name_es, name_ca, name_it)
+            VALUES
+                ("PublicEntities", "Ivoz\Provider\Domain\Model\PublicEntity\PublicEntity", 1, 1, 0, "Entity", "Entidad", "Entidad", "Entity")'
```

Comment:
This probably requires doble backlash \\  check after applying the delta

Reply:
fixed :+1: